packages/types/src/json.ts

```diff
@@ -218,7 +248,11 @@ export interface UserJSON extends ClerkResourceJSON {
   phone_numbers: PhoneNumberJSON[];
   web3_wallets: Web3WalletJSON[];
   external_accounts: ExternalAccountJSON[];
+  enterprise_accounts: EnterpriseAccountJSON[];
   passkeys: PasskeyJSON[];
+  /**
+   * @deprecated use `enterprise_accounts` instead
+   */
   saml_accounts: SamlAccountJSON[];
```

Comment:
FAPI is going to continue returning `saml_accounts`, to be backwards compatibility, however from now on newer clerk-js versions we should implement against `enterprise_accounts`.